### DISCUSSION TITLE
How can I download a previous version of resposively?

### DISCUSSION THREAD

**Discussion by x-nrdev** [responsively-org/responsively-app#1011] [Q&A]
Hi I would like to know how to download a previous version of Responsively because I usually use this app to manage old websites and the old version doesn't break the layout of these websites which the new one does

**Comment by manojVivek** [comment]
Hi @nrdevpy, Can you please let us know what's breaking in the newer version so that we can fix it?

**Reply by x-nrdev** [reply]
Hi @manojVivek, when I downloaded the app in my PC apparently it works, but with the laptop that I normally use to work, I usually use responsively to manage websites, these websites are not literally "responsive", it has a mechanism which detects the device to show the user specific layouts.

Previously to Responsively 1.5 I also was using this app and it works properly but for some reason on the 1.5 version the app isn't able to detects the device you're using to see the website then the web breaks.

Anyway now that I tested on my personal PC this new version apparently it works, I will let you know If I have any issue.

Thank you very much for the fast answer.
Have a nice day 👍🏿.
